Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,41 @@
 import React from 'react';
 import './App.css';
 
+//import buttons
+import ClearBtn from './components/ButtonComponents/ActionButton'
+import Buttons from './components/ButtonComponents/NumberButton'
+import Display from './components/DisplayComponents/CalculatorDisplay'
+
+
+
+//inputs for the buttons
+let input = [ 
+  {value: [1,2,3,4,5,6,7,8,9,0]},
+  {operand: ['/','*','+','-']}
+]
+
 const App = () => {
   return (
     <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+      <div>
+        <h3>Welcome to React Calculator</h3>
+        <p>
+          We have given you a starter project. You'll want to build out your
+          components in their respective files, remove this code and replace it
+          with the proper components.
+        </p>
+        <p>
+          <strong>
+            Don't forget to `default export` your components and import them here
+            inside of this file in order to make them work.
+          </strong>
+        </p>
+      </div>
+      <div>
+        <Display />
+        <ClearBtn />
+        <Buttons />
```

Comment:
this is a good start, and but you need to start passing variables into your child components in order to use them in lower components.
also you need to find a way to dynamically run that array into your Buttons component to create all the buttons.
you can type this in manual or run it thorugh a .map funciton.